packages/quantic/force-app/main/default/lwc/quanticSort/quanticSort.js

```diff
@@ -65,6 +76,42 @@ export default class QuanticSort extends LightningElement {
 
   connectedCallback() {
     registerComponentForInit(this, this.engineId);
+    getBueno(this).then(() => {
+      this.customSortOptions.forEach((option) => {
+        const criterion = option.criterion;
+        // Checks that the label of the custom sort option is a string
+        if (!option.label || !Bueno.isString(option.label)) {
+          console.error(
+            `The ${this.template.host.localName} component requires a label to be specified.`
+          );
+          this.setSortOptionsConfigurationError();
+        }
+        // Checks that the value of the custom sort option is a string
+        if (!option.value || !Bueno.isString(option.value)) {
+          console.error(
+            `The ${this.template.host.localName} component requires a value to be specified.`
+          );
+          this.setSortOptionsConfigurationError();
+        }
+        /**
+         * Checks that the criterion of the custom sort option is an object which respects the following:
+         * By should be a string
+         * If sorting by field, field must be a string
+         * If sorting by field or date, order must be a string
+         */
+        if (
+          !Bueno.isString(criterion.by) ||
+          (criterion.by === 'field' && !Bueno.isString(criterion.field)) ||
+          ((criterion.by === 'field' || criterion.by === 'date') &&
+            !Bueno.isString(criterion.order))
```

Comment:
Check if Bueno has the ability to validate an object according to a given schema instead of validating each property at once.